api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -152,17 +159,46 @@ const (
 
 const KubeObjectProtectionCaptureIntervalDefault time.Duration = 5 * time.Minute
 
+type ResourceCaptureInstance struct {
+	Name string `json:"name,omitempty"`
+
+	//+optional
+	IncludeClusterScopedResources bool `json:"includeClusterScopedResources,omitempty"`
+
+	//+optional
+	IncludedResources []string `json:"includedResources,omitempty"`
+
+	//+optional
+	ExcludedResources []string `json:"excludedResources,omitempty"`
+
+	//+optional
+	LabelSelector metav1.LabelSelector `json:"labelSelector,omitempty"`
+}
+
+type ResourceRestoreInstance struct {
+	BackupName string `json:"backupName,omitempty"`
+
+	//+optional
+	IncludeClusterScopedResources bool `json:"includeClusterScopedResources,omitempty"`
+
+	//+optional
+	IncludedResources []string `json:"includedResources,omitempty"`
+
+	//+optional
+	ExcludedResources []string `json:"excludedResources,omitempty"`
+}
+
 type KubeObjectProtectionSpec struct {
 	// Preferred time between captures
 	// +optional
 	// +kubebuilder:validation:Format=duration
 	CaptureInterval *metav1.Duration `json:"captureInterval,omitempty"`
 
 	//+optional
-	ResourceCaptureOrder [][]string `json:"resourceBackupOrder,omitempty"`
+	ResourceCaptureOrder []ResourceCaptureInstance `json:"resourceCaptureOrder,omitempty"`
```

Comment:
I'm thinking add `Groups` for alignment with slice type: `ResourceCaptureGroupsOrder`

Reply:
This name didn't change, but I think it is actually good, not sure the `Groups` helps.